react/week4/github-users-search-app/src/components/userContext.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
Smaller thing but I'd usually keep my contexts separate from the components in another folder e.g. a "contexts", "lib" or "utils" folder with my various contexts

I'd actually take a lot of the useState values you have in your App.js and have it here in this context file and export the wrapping Provider itself and pass {children} as a prop into it. So when you pass the "value" in App.js I would keep the brain logic/useState values via the context.

export Provider <Context.Provider value={/* some value */}>{children}</Context.Provider>